lambda/cfn.yaml

```diff
@@ -292,7 +292,7 @@ Resources:
   PriceMigrationEngineSalesforcePriceCreationLambda:
     Type: AWS::Lambda::Function
     Properties:
-      Description: Lambda used to create Price_Rise__c objects in salesforce realting to the price rise.
+      Description: Lambda used to create Price_Rise__c objects in salesforce relating to the price rise.
```

Comment:
Have to make a change to a resource to get cloudformation to pick up any new exports.
ie. only changes in resources act as a trigger but then exports get detected as well.
Otherwise exports just get ignored.